lib/keymap-helpers.nix

```diff
@@ -22,6 +22,8 @@ rec {
     remap = nixvimOptions.defaultNullOpts.mkBool false "Make the mapping recursive. Inverses \"noremap\"";
 
     desc = nixvimOptions.mkNullOrOption types.str "A textual description of this keybind, to be shown in which-key, if you have it.";
+
+    buffer = nixvimOptions.defaultNullOpts.mkBool false "Make the mapping buffer-local. Equivalent to adding <buffer> to a map.";
```

Comment:
```suggestion
    buffer = nixvimOptions.defaultNullOpts.mkBool false "Make the mapping buffer-local. Equivalent to adding `<buffer>` to a map.";
```

Reply:
There are no backticks around <silent>, <script>, etc. in the strings above

Reply:
Fair point. There probably should be, but that should be done in a separate commit.

Personally I don't mind if that's tackled in this PR or not. Otherwise, I'm happy for this option to be consistent with the others and not use backticks for now.

Reply:
Either way